client/src/components/QuizPage/index.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react"
+import Button from "../sharedComponent/Button"
+import BackButton from "../sharedComponent/BackButton"
+import Card from "../Card"
+import ProgressBar from "../sharedComponent/ProgressBar"
+import CircleProgressBar from "../sharedComponent/CircleProgressBar/"
+import qustions from "../../Questions"
+class Quiz extends Component {
+  state = {
+    counter: 1,
+    options: "",
+    score: [],
+    percent: 0
+  }
+  clicked = ({ target }) => {
+    const { name } = target
+    const { score } = this.state
+    score[this.state.counter - 1] = parseInt(name)
+    this.setState({
+      score: score,
+      counter: this.state.counter + 1
+    })
+    if (this.state.counter === 9) {
+      this.setState({ percent: 50 })
+    }
+    if (this.state.counter === 18) {
+      window.location.href = `/dd${this.state.score}`
+    }
+  }
+  componentDidMount() {
+    this.setState({
+      options: (
+        <>
+          <Button
+            borderRadius="30px"
+            Desktopmargin="1%"
+            Desktoppadding="28px 15px"
+            fontSize="28px"
+            width={"45%"}
+            name={0}
+            onClick={this.clicked}
+          >
+            Never
+          </Button>
+          <Button
+            borderRadius="30px"
+            Desktopmargin="1%"
+            Desktoppadding="28px 15px"
+            fontSize="28px"
+            width={"45%"}
+            name={1}
+            onClick={this.clicked}
+          >
+            Reraly
```

Comment:
this is a typo here.
```suggestion
            Rarely
```